dbreps2/src/general/blankpages.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use dbreps2::{Frequency, Report};
+use mysql_async::{prelude::Queryable, Conn};
+
+pub struct Row {
+    namespace: u32,
+    title: String,
+}
+
+pub struct BlankPages;
+
+#[async_trait::async_trait]
+impl Report<Row> for BlankPages {
+    fn title(&self) -> &'static str {
+        "Blank single-author pages"
+    }
+
+    fn query(&self) -> &'static str {
+        "
+        SELECT
+          page_namespace,
+          CONVERT(page_title USING utf8)
```

Comment:
We don't need the CONVERT(...) anymore, just selecting a plain page_title works fine.